lib/core/src/irods_gsi_object.cpp

```diff
@@ -29,22 +29,20 @@ namespace irods {
                 _interface.c_str()));
         }
 
-        auth_ptr auth;
-        if (const auto err = auth_mgr.resolve(AUTH_GSI_SCHEME, auth); !err.ok()) {
-            return PASSMSG("Failed to resolve the GSI auth plugin.", err);
+        const auto& type = irods::AUTH_GSI_SCHEME;
+
+        auth_ptr ap;
+        if (const auto resolve_err = auth_mgr.resolve(type, ap); !resolve_err.ok()) {
```

Comment:
You can reuse the error object 8n the inner if-stmt if the const qualifier is removed.

Not required by any means. This is totally fine as it is.

Reply:
Will do